MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/APICall.cs

```diff
@@ -15,7 +15,7 @@ public class APICall
         private RestClient _restClient;
         public APICall()
         {
-            _apiKey = Environment.GetEnvironmentVariable("API_KEY");
+            _apiKey = ConfigurationManager.AppSettings["APIKey"];
```

Comment:
I think this was changed to allow it to work on Heroku, right - did you mean to change it back or was this an accident?